docs/configuration/system.md

```diff
@@ -206,14 +206,14 @@ If `STORAGE_BACKEND` is not defined, this setting will be ignored.
 
 ## TIME_ZONE
 
-Default: UTC
+Default: `UTC`
```

Comment:
It might help to represent this explicitly as a string to avoid confusion with a global constant.
```suggestion
Default: `"UTC"`
```

Reply:
@jeremystretch thanks. Change made